lib/puma/cluster.rb

```diff
@@ -75,9 +77,13 @@ def initialize(idx, pid, phase, options)
         @last_checkin = Time.now
         @last_status = '{}'
         @term = false
+        @socket = socket
+        @capacity = 0
```

Comment:
Is there a reason we can't re-use information from the threadpool, I wonder, rather than setting a separate counter?